packages/admin/src/targetGroups/TargetGroupsGrid.tsx

```diff
@@ -263,12 +263,11 @@ export function TargetGroupsGrid({
         },
     ];
 
-    const { filter: gqlFilter, search: gqlSearch } = muiGridFilterToGql(columns, dataGridProps.filterModel);
+    const { search: gqlSearch } = muiGridFilterToGql(columns, dataGridProps.filterModel);
```

Comment:
why did the filters get removed here?
the targetgroup grid has filters right?

Reply:
Initially I wanted to filter for the test contact list here, not in the api. After undoing my changes, I forgot to add the filters here again. I'm sorry! 
I corrected it here: fd436ca3cf5abca28f238d5af16f035623c85606